src/js/api/shipment.js

```diff
@@ -0,0 +1,259 @@
+ *  - 'delivery_date' - The date the shipment began.
+ *  - 'origin_country' - The 'ISO 3166-2' country code where the shipment begins.
+ *  - 'origin_city' - The city where the shipment begins.
+ *  - 'destination_country' - The 'ISO 3166-2' country code where the shipment ends.
+ *  - 'destination_city' - The city where the shipment ends.
+ *  - 'attachments' - A list of RIPE Core attachment IDs.
+ * @returns {Promise} The shipment's data.
+ */
+ripe.Ripe.prototype.createShipmentP = function(options, callback) {
+    return new Promise((resolve, reject) => {
+        this.createShipment(options, (result, isValid, request) => {
+            isValid ? resolve(result) : reject(new ripe.RemoteError(request, null, result));
+        });
+    });
+};
+
+/**
+ * Updates a shipment on Ripe Core.
+ *
+ * @param {Number} number The number of the shipment to find by.
+ * @param {Object} options An object with options, such as:
+ *  - 'status' - The shipment status.
+ *  - 'order' - The order associated with this shipment.
+ *  - 'bulk_order' - The bulk order associated with this shipment.
+ *  - 'courier' - The courier for this shipment.
+ *  - 'tracking_number' - The tracking number associated with this shipment.
+ *  - 'tracking_url' - The tracking url associated with this shipment.
+ *  - 'shipping_date' - The date the shipment ended.
+ *  - 'delivery_date' - The date the shipment began.
+ *  - 'origin_country' - The 'ISO 3166-2' country code where the shipment begins.
+ *  - 'origin_city' - The city where the shipment begins.
+ *  - 'destination_country' - The 'ISO 3166-2' country code where the shipment ends.
+ *  - 'destination_city' - The city where the shipment ends.
+ *  - 'attachments' - A list of RIPE Core attachment IDs.
+ * @param {Function} callback Function with the result of the request.
+ * @returns {XMLHttpRequest} The XMLHttpRequest instance of the API request.
+ */
+ripe.Ripe.prototype.updateShipment = function(number, options, callback) {
+    callback = typeof options === "function" ? options : callback;
+    options = typeof options === "function" || options === undefined ? {} : options;
+    const url = `${this.url}shipments/${number}`;
+    options = Object.assign(options, {
+        url: url,
+        method: "PUT",
+        params: options,
+        auth: true
+    });
+    options = this._build(options);
+    return this._cacheURL(options.url, options, callback);
+};
+
+/**
+ * Updates a shipment on Ripe Core.
+ *
+ * @param {Number} number The number of the shipment to find by.
+ * @param {Object} options An object with options, such as:
+ *  - 'status' - The shipment status.
+ *  - 'order' - The order associated with this shipment.
+ *  - 'bulk_order' - The bulk order associated with this shipment.
+ *  - 'courier' - The courier for this shipment.
+ *  - 'tracking_number' - The tracking number associated with this shipment.
+ *  - 'tracking_url' - The tracking url associated with this shipment.

```

Comment:
Please replicate in the other documentation in this file
```suggestion
 *  - 'tracking_url' - The tracking URL associated with this shipment.

```